src/content/overview/observability/observability-platform-api/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Observability Platform API"
+linkTitle: Observability Platform API
+description: Documentation on the observability-platform API architecture deployed and maintained by Giant Swarm.
+weight: 80
+menu:
+  principal:
+    identifier: overview-observability-platform-api
+    parent: overview-observability
+user_questions:
+  - What is the observability-platform API?
+  - What is the observability-platform API for?
+  - How to use the observability-platform API?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2025-02-10
+---
+
+The following page describes the concept of the Observability Platform API
+
+## What it is
+
+The observability platform ingests and allows to explore system and application observability data from inside Giant Swarm managed clusters by default.
+
+The `observability platform API` opens up the observability platform to be used from the outside - which means any resource not managed by Giant Swarm. You can ingest observability data from any source by sending them to the `observability platform API`. Additionally the API allows to query observability data from wherever you want.
+
+With this you can ingest observability data from, for example, a SaaS-Database that might be a dependency of one of your workloads on a Giant Swarm managed cluster. Or you could explore the observability platforms data from your own, remote Grafana instance or any other observability tooling.
+
+**Note:** At this point the `observability platform API` only allows the ingestion of logs and events. The ingestion of metrics will follow in a later release. Keep an eye on our [changes and releases]({{< relref "/changes/observability-platform/" >}}) or this document for updates.
+
+The `observability platform API`s main objectives are to:
+
+* enable you to securely ingest and access observability data in our observability platform from anywhere - also from outside of Giant Swarm managed clusters.
+* sanitize (sampling, relabelling, etc.) observability data to align with our general data standards in the observability platform.
+* support for the OpenTelemetry Protocol (OTLP).
```

Comment:
I don't get the last point; what do you mean? Our API is implementing OTLP?

Reply:
Basically the API supports OTLP for logs thanks to us using alloy as a "gateway" in the write path so customers can send OTLP data to our backend.